lib/core/src/server/templates/index.html.ejs

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta content="IE=edge" http-equiv="X-UA-Compatible" />
+    <title>Storybook</title>
+    <link rel="shortcut icon" href="favicon.ico?v=1" />
+
+    <% if (options.links) { %>
```

Comment:
@ndelangen I'm trying to figure out how to plug into this for CSS files we extract out with `mini-css-extract-plugin`, are there any pointers? Been digging for clues for the past couple of days.

Reply:
@diagramatics I'm happy to help out, do you have a repo I can use to debug this?

Reply:
Thanks! I have a repro on `cra-kitchen-sink` in https://github.com/diagramatics/storybook/tree/repro-mini-css-extract-plugin, with the Webpack config extending the default config and replacing the `style-loader` with `mini-css-extract-plugin`.

You can replace `mini-css-extract-plugin` with `extract-text-webpack-plugin` to make it in parity with what `create-react-app` has, but essentially if I understand correctly the issue is that `GeneratePagePlugin` (and in extension, the Webpack config files to add the plugin in `lib/core/src/server/config`) doesn't have a way to hook into these common CSS file extraction Webpack plugins to output them into this particular section of the `index.html.ejs` template (`options.links`).

https://github.com/storybooks/storybook/blob/013822a39e5ca857efa991a4e67cc3c3c862777b/lib/core/src/server/config/webpack.config.dev.js#L45-L59

Notice there's no `options.links` specified. Not to mention there is no way we can pass generated CSS files in since this is not in the middle of any hook execution such as `additionalChunkAssets` to inject the files in.

This is as much as I get digging around the codebase, so let me know if you need more info (or if I'm absolutely wrong 😅). I'm also on Slack by the same username.